modules/ximgproc/samples/radon_transform_demo.cpp

```diff
@@ -0,0 +1,16 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include <opencv2/highgui.hpp>
+#include <opencv2/ximgproc/radon_transform.hpp>
+
+int main() {
+	cv::Mat src = cv::imread("peilin_plane.png", cv::IMREAD_GRAYSCALE);
```

Comment:
> cv::

In OpenCV samples we add `using namespace cv;` after includes and removing all `cv::` prefixes.